config/config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "GCLOUD_PROJECT": "google.com:gulliver",
-  "CLOUD_BUCKET": "gulliver.google.com.a.appspot.com",
-  "CLIENT_ID": "605287872172-js6omne47i1k79hnfo7d4bdu9rlemslr.apps.googleusercontent.com",
+  "GCLOUD_PROJECT": "run `gcloud config get-value project`",
+  "CLOUD_BUCKET": "your cloud bucket",
+  "CLIENT_ID": "your client id",
```

Comment:
Yeah good point, was thinking there was no concise way of explaining this but maybe `see https://console.cloud.google.com/storage/browser?project=$GCLOUD_PROJECT` is an improvement?